lib/active_storage_validations/analyzer/image_analyzer/vips.rb

```diff
@@ -47,6 +40,13 @@ def image_from_path(path)
       end
     end
 
+    def supported?
+      require "vips"
+      true
+    rescue LoadError
+      false
```

Comment:
Why did you remove the logger.info "..." line? Plus you left in mini_magick analyzer

Reply:
Whoops! Accidental 😅